packages/frontend-app/src/app/components/Card/index.stories.tsx

```diff
@@ -0,0 +1,30 @@
+import React from "react";
+import { Meta, StoryObj } from "@storybook/react";
+
+import { Card } from "./index";
+import { BrowserRouter as Router } from "react-router-dom";
+
+const meta: Meta<typeof Card> = {
+  title: "Card",
+  component: Card,
+  decorators: [
+    (Story) => (
+      <div className="bg-black">
+        <Router>
+          <Story />
+        </Router>
+      </div>
```

Comment:
```suggestion
      <div className="bg-black w-screen h-screen fixed top-0 left-0 flex items-center justify-center">
        <Router>
          <div className="w-96">
              <Story />
          </div>
        </Router>
      </div>
```
We need to position the Card properly within Storybook